pynumaflow/function/udfunction_pb2.pyi

```diff
@@ -0,0 +1,55 @@
+from google.protobuf import descriptor as _descriptor
```

Comment:
Is this also generated?

Reply:
this is not generated, she made it by hand